### PR TITLE
Documentation setup

### PR DESCRIPTION
**Pull request by frederikgeth**
Experimenting with the basic setup, mainly to be able to add latex model descriptions and generate pdfs.

EDIT: not sure if PDF generation worked, but otherwise documentation compilation looks great.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,3 +12,7 @@ HarmonicPowerModels.jl is an extension package of PowerModels.jl for Haromonic (
 
 ## Acknowledgements
 This code has been developed at BASF Antwerp and CSIRO. The primary developers are Tom Van Acker ([@timmyfaraday](https://github.com/timmyfaraday)) and Frederik Geth ([@frederikgeth](https://github.com/frederikgeth)).
+
+
+## License
+This code is provided under a MIT license.
```
Comment (timmyfaraday) [inline README.md]: This has been changed to a BSD 3-Clause License.

**Comment by frederikgeth** [comment]
ok, documentation definitely broken - trying to fix

EDIT: everything should be working now. much excite

**Comment by timmyfaraday** [comment]
I have added the public key and secret to github, and enabled the GitHub pages. 
Should be tested

File: docs/src/index.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# HarmonicPowerModels.jl Documentation
+
+```@meta
+CurrentModule = HarmonicPowerModels
+```
```
Comment (timmyfaraday) [inline docs/src/index.md:5]: Is this necessary?

File: docs/src/index.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+HarmonicPowerModels.jl is a research-grade Julia/JuMP package for experimentation with Steady-State Power Network Optimization with Harmonics, extending PowerModels.jl.
+
+
+## Installation
+
+For now, HarmonicPowerModels is unregistered. Nevertheless, you can install it through
+
+```
+] add https://github.com/timmyfaraday/HarmonicPowerModels.jl.git
```
Comment (timmyfaraday) [inline docs/src/index.md:17]: .git necessary?

File: docs/make.jl [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+const _PAGES = [
+    "Home" => "index.md",
+];
+
+@time Documenter.makedocs(
+    modules = [HarmonicPowerModels],
+    format = Documenter.HTML(mathengine = Documenter.MathJax()),
+    sitename = "HarmonicPowerModels",
+    authors = "Tom Van Acker and contributors.",
```
Comment (timmyfaraday) [inline docs/make.jl]: `Tom Van Acker, Frederik Geth and contributors.` seems more accurate!!!

**Review by timmyfaraday — COMMENTED** [review]
Small changes

File: README.md (hunk 1 above)
Comment (frederikgeth) [reply README.md]: updated branch from master, resolved

File: docs/src/index.md (hunk 2 above)
Comment (frederikgeth) [reply docs/src/index.md:5]: I think so

File: docs/src/index.md (hunk 3 above)
Comment (frederikgeth) [reply docs/src/index.md:17]: yes, I believe this is recommended

File: docs/make.jl (hunk 4 above)
Comment (frederikgeth) [reply docs/make.jl]: done

**Pull request merged**